modules/custom/d_social_media/js/last-element-in-a-row.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
Wrap this code in JS behavior - https://www.lullabot.com/articles/understanding-javascript-behaviors-in-drupal